infra/lib/log-source.ts

```diff
@@ -72,9 +72,13 @@ export interface LogSourceConfig {
     expand_records_from_payload?: string;
     compression: string;
     s3_source?: {
+      enabled?: boolean,
```

Comment:
I don't think we need enabled for s3 source since is doesn't create a resource per log source (it just maps conceptually to a prefix in the single sources bucket).